fe/lib/components/JobCategorySuggest/JobCategorySuggest.tsx

```diff
@@ -22,7 +21,8 @@ type RadioProps = ComponentPropsWithRef<typeof RadioGroup>;
 
 export interface JobCategorySuggestProps
   extends Partial<Omit<RadioProps, 'id'>> {
-  positionProfile: JobCategorySuggestionPositionProfileInput;
```

Comment:
I'm guessing by not using this input anymore this will mean we'll no longer support passing in a list of [`positionFormattedDescriptions`](https://developer.seek.com/schema/#/named-type/JobCategorySuggestionPositionProfileInput/field/positionFormattedDescriptions) and [`positionOrganizations`](https://developer.seek.com/schema/#/named-type/JobCategorySuggestionPositionProfileInput/field/positionOrganizations). Are we comfortable with that?

Reply:
Where were those being used?

Reply:
Assuming this `JobCategorySuggestionPositionProfileInput` type matches our schema: https://developer.seek.com/schema/#/named-type/JobCategorySuggestionPositionProfileInput

Reply:
Ryanair uses it as well:
https://github.com/SEEK-Jobs/indie-ryanair-frontend/blob/73869d4002ea207475494db42540370bdc3d02d1/src/pages/Posting/jobCategories/components/JobCategorySuggestForm.tsx#L53-L69

Reply:
Alright, going back to the original question, there are two issues to consider. One is that RyanAir will possibly get less accurate category suggestions with this new simplified interface, and the other is that partners looking to wingman for guidance may implement something that provides less accurate suggestions also. The two paths that I can see from here are to decide not worry about those issues and merge this as is, or add the missing fields to the interface as optional.

What do we reckon?

Reply:
This interface is the only way to use AIPS’s category predict interface. This is more of the Back Office having very basic needs than Ryanair/partners having special needs. We definitely need to keep supporting it.

The third option is to use `React.memo` with a custom equality function:
<https://reactjs.org/docs/react-api.html#reactmemo>

Reply:
I don't like the idea of adding more props to this component, because I think it looks complex. Happy to explore React.memo deeper. Are you seeing that becoming part of the implementation of this JobCategorySuggest component, or back at the call site in the backoffice?

Reply:
We should place it in Wingman - it’s the components responsibility to keep its state sane when its inputs change